### PR TITLE
Stock Class and symbols file

### PR DESCRIPTION
**Pull request by makni-mehdi**
GUI team, is this fine to you?

### CODE REVIEW

File: src/components/stock.h [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#ifndef STOCK_H
+#define STOCK_H
+
+#include <string>
+#include <QJsonObject>
+
+class Stock {
+private:
+  std::string symbol;
+  QJsonObject dataByDay;
```
Comment (guruprerana) [inline src/components/stock.h:10]: Would it be possible to add a short comment with an example of how this JSON object looks like?

File: src/components/stock.h [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  QJsonObject dataByMinute;
+  long latestTimeStampByDay;
+  long latestTimeStampByMinute;
+
+public:
+  Stock(std::string);
+  ~Stock();
+
```
Comment (guruprerana) [inline src/components/stock.h:19]: What do you think about adding a function `bool isValid()` which tells you whether if the stock symbol is valid and there is some data on it?

File: src/components/stock.h (hunk 1 above)
Comment (makni-mehdi) [reply src/components/stock.h:10]: Alrighty! will do it now

File: src/components/stock.h (hunk 2 above)
Comment (makni-mehdi) [reply src/components/stock.h:19]: Good catch! Yup

**Pull request merged**